### PR TITLE
Register member endpoint integration

### PR DESCRIPTION
**Pull request by MeBrei**
Implements actual member registration and replaces the org member list mock. It also adds a new control endpoint to add additional users so that adding members can be tested in cypress.

Related to #349

### CODE REVIEW

File: proxy/src/registry/transaction.rs [hunk 1]
```diff
@@ -109,6 +109,14 @@ pub enum Message {
         /// Identity id originated from librad.
         id: Option<String>,
     },
+
+    /// Issue a member registration for a given handle under a given org.
+    MemberRegistration {
```
Comment (xla) [inline proxy/src/registry/transaction.rs:114]: We should force consistent camelcasing by using `#[serde(rename_all = "camelCase")]` on the variants as well.

Comment (MeBrei) [reply proxy/src/registry/transaction.rs:114]: Does [L83](https://github.com/radicle-dev/radicle-upstream/blob/0f5dd37f97d869dc5f5a7cff2d75df8197d2b953/proxy/src/registry/transaction.rs#L83) not affect the variants as well?

Comment (xla) [reply proxy/src/registry/transaction.rs:114]: No unfortunately it doesn't as serde treats every struct variant as its own container.

Comment (MeBrei) [reply proxy/src/registry/transaction.rs:114]: addressed in #449

**Pull request merged**